elements/jsonform/src/main.js

```diff
@@ -146,9 +143,9 @@ export class EOxJSONForm extends LitElement {
     });
   }
 
-  async firstUpdated() {
-    this.setSchema(await this.parseProperty(this.schema));
-    this.setDefaultValues(await this.parseProperty(this.startVals));
+  async updated() {
+    this.schema = await this.parseProperty(this.schema);
```

Comment:
If I understand correctly, the `updated()` function triggers every time the component is updated and/or re-rendered. Maybe it would make sense to check if the schema is already set? Something like if the current schema is already the same as the property, do nothing. Same for startVals.